templates/zerver/no_auth_backend.html

```diff
@@ -0,0 +1,10 @@
+{% extends "zerver/portico.html" %}
+{% block portico_content %}
+
+<h1>No Auth Backend Enabled</h1>
+
+<p>
+    TODO put instructions on how to set up an auth backend here.
+</p>
+
+{% endblock %}
```

Comment:
I think it might be better for the login template to show the error, rather than sending the user to a separate page, especially one like this which will load and show the error message even on a properly configured server.